lib/cfn-nag/custom_rules/RDSInstanceBackupRetentionPeriodRule.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'cfn-nag/violation'
+require_relative 'base'
+
+class RDSInstanceBackupRetentionPeriodRule < BaseRule
+  def rule_text
+    'RDS instance should have backup retention period greater than 0'
+  end
+
+  def rule_type
+    Violation::WARNING
+  end
+
+  def rule_id
+    'W75'
+  end
+
+  def audit_impl(cfn_model)
+    rds_dbinstances = cfn_model.resources_by_type('AWS::RDS::DBInstance')
+
+    violating_rdsinstances = rds_dbinstances.select do |instance|
+      instance.backupRetentionPeriod && instance.backupRetentionPeriod == 0
```

Comment:
even though the cfn spec says this property is an int.... i've seen cfn exhibit "weak typing" and accept string versions of numbers.  so in this case, the string "0" wouldn't be a violation.  can you confirm the behavior or do a to_i/to_s to normalize?  thx!

Reply:
Fixed in https://github.com/stelligent/cfn_nag/pull/438/commits/2973208155f28e94cd777faa571ba1beb899a417